django/contrib/postgres/fields/serial.py

```diff
@@ -0,0 +1,61 @@
+from django.db import connection
+from django.db.models import Field
+from django.utils.translation import ugettext_lazy as _
+
+
+class SerialField(Field):
```

Comment:
In this case you'll have to add some boilerplate and tests to make sure a `ForeignKey` pointing a unique `SerialField` generates the correct remote field and datatype. This includes at the model and migration level. You'll also have to add cleaning tests to make sure submitting a non integer or an overflowing one generates the appropriate `ValidationError`. I really think subclassing `PositiveIntegerField` would be more appropriate than duplicating 90% of its logic here.